CONTRIBUTING.md

```diff
@@ -1,30 +1,151 @@
+
+The `footer` is optional. The [Commit Message Footer](#commit-footer) format describes what the footer is used for and the structure it must have.
+
+#### <a name="commit-header"></a>Commit Message Header
+
+```
+<type>(<scope>): <short summary>
+  │       │             │
+  │       │             └─⫸ Summary in present tense. Sentence case. No period at the end.
+  │       │
+  │       └─⫸ Commit Scope: design-tokens|icons|web|web-react|web-twig|exporter-css|exporter-svg|
+  │                          ci|repo
+  │
+  └─⫸ Commit Type: Feat|Fix|Perf|Revert|Docs|Style|Refactor|Test|Chore|Deps
+```
+
+The `<type>` and `<summary>` fields are mandatory, the `(<scope>)` field is optional.
+
+##### Type
+
+Must be one of the following:
+
+- **Chore**: Changes to our CI configuration files and scripts (examples: CircleCi, SauceLabs) or changes that affect build system
+- **Deps**: Changes to dependencies
+- **Docs**: Documentation only changes
+- **Feat**: A new feature
+- **Fix**: A bug fix
+- **Perf**: A code change that improves performance
+- **Refactor**: A code change that neither fixes a bug nor adds a feature
+- **Revert**: Reverting code changes
+- **Test**: Adding missing tests or correcting existing tests
+- **Style**: Styling of components of changes to code style
+
+##### Scope
+
+The scope should be the name of the npm package affected (as perceived by the person reading the changelog generated from commit messages).
+
+The following is the list of supported scopes:
+
+- `design-tokens`
+- `icons`
+- `web`
+- `web-react`
+- `web-twig`
+- `exporter-css`
+- `exporter-svg`
+- `ci`
+- `repo`
+
+There are currently a few exceptions to the "use package name" rule:
+
+- `ci`: used for changes that affects Continuous Integration process and builds
+
+- `repo`: used for repository wide changes
+
+- none/empty string: useful for `test` and `refactor` changes that are done across all packages (e.g. `Test: Add missing unit tests`) and for docs changes that are not related to a specific package (e.g. `Docs: Fix typo in tutorial`).
+
+##### Summary
+
+Use the summary field to provide a succinct description of the change:
+
+- use the imperative, present tense: "change" not "changed" nor "changes"
+- use the sentence case (capitalize the first letter of the sentence)
+- no dot (.) at the end
+
+#### <a name="commit-body"></a>Commit Message Body
+
+Just as in the summary, use the imperative, present tense: "fix" not "fixed" nor "fixes".
+
+Explain the motivation for the change in the commit message body. This commit message should explain _why_ you are making the change.
+You can include a comparison of the previous behavior with the new behavior in order to illustrate the impact of the change.
+
+#### <a name="commit-footer"></a>Commit Message Footer
+
+The footer can contain information about breaking changes and deprecations and is also the place to reference GitHub issues, Jira tickets, and other PRs that this commit closes or is related to.
```

Comment:
This looks like a breaking change to our breaking change rules 🙂. Can we still use the `BREAKING CHANGE()` type? Because it's not listed in allowed types a few sections above.

Reply:
https://github.com/lmc-eu/code-quality-tools/blob/main/packages/conventional-changelog-lmc/src/bump-opts.js#L13-L16
https://github.com/lmc-eu/code-quality-tools/blob/main/packages/conventional-changelog-lmc-github/tests/index.test.js#L26-L30

Currently, the BREAKING CHANGE pattern can be in the header or footer and it will work ;-)